rules/android_local_test/impl.bzl

```diff
@@ -383,7 +383,7 @@ def finalize(
         coverage_common.instrumented_files_info(
             ctx = ctx,
             source_attributes = ["srcs"],
-            dependency_attributes = ["deps", "runtime_deps", "data"],
```

Comment:
Alternative solution here is to break this out into it's own `coverage_ctx` so that we can override it easier when assembling the custom processing pipeline from `rules_kotlin`. The `android_library` rule does this already.